server/routes/all.ts

```diff
@@ -39,6 +40,7 @@ export default function routes(userService: UserService): Router {
   router.use('/analytics/:pgdRegionCode([A-Z0-9]{2,5}|National)?', analyticsRouter(standardRouter(userService)))
   router.use('/prisoner-images/:prisonerNumber.jpeg', prisonerImagesRoutes(imageRouter()))
   router.use('/throw-test-error', throwTestErrorRouter(standardRouter(userService)))
+  router.use('/notification/dismiss', notificationRouter(standardRouter(userService)))
```

Comment:
in case we add another notification somewhere, i think it'd be good to make the URL a bit more unique to _this_ notification…?